pennylane/hf/tapering.py

```diff
@@ -15,20 +15,25 @@
 This module contains the functions needed for tapering qubits using symmetries.
 """
 # pylint: disable=unnecessary-lambda
-import functools
 
+import functools
+import numpy
 import autograd.numpy as anp
 import pennylane as qml
 from pennylane import numpy as np
+from pennylane.hf.hamiltonian import _generate_qubit_operator
 
 
-def _binary_matrix(terms, num_qubits):
+def _binary_matrix(terms, num_qubits, wire_map=None):
```

Comment:
What is the purpose of this change? Is it needed for tapering the HF state?

Reply:
I realized that `_binary_matrix` will require to have wire mapping when dealing with custom wire labels and discontinuous wire labels.